conf_linux_public.py

```diff
@@ -89,9 +88,14 @@ def get_building_cmd(command, gcc_latest, enable_devtoolset):
     else:
         return f'{enable_devtoolset} && {command}' #enable new compiler on CentOS
 
+# Choose repository in accordance with prefix of product type
+if product_type.startswith("public"):
+    repo_name = 'MediaSDK'
+else:
```

Comment:
it is better to check next gen type too and rise exception in other cases
or next gen should not be default choice...